### PR TITLE
MWPW-153658 audit script

### PR DESCRIPTION
**Pull request by npeltier**
- parse one or several url or sitemap files (can be added in manifest),
- for each url from sitemap, parse page for personnalization fragment or fragment,
- for each ost link found in html, keep usage,
- for each osi, get wcs entry,
- output everything as CSV

Test URLs:
- Before: https://main--mas--adobecom.hlx.live/
- After: https://MWPW-153658--mas--npeltier.hlx.live/

### CODE REVIEW

File: misc/audit.mjs [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+            }
+        });
+    }
+    return result;
+}
+
+const prefixWcsKey = (key) => ('wcs ' + key);
+
+async function getCommerceData(osi) {
+    const response = await fetch(wcsUrl(osi));
```
Comment (yesil) [inline misc/audit.mjs:127]: Is that okay if the OSI doesn't return an offer anymore via WCS?  AOS can always deliver the parameters of an OSI.

**Review by yesil — APPROVED** [review]
It looks very good to me.

In https://github.com/adobecom/mas/pull/22, there is eslint/prettier lint rules. If possible I suggest to lint your code with it before the initial merge.

File: misc/audit-manifest.txt [hunk 2]
```diff
@@ -0,0 +1,186 @@
+https://www.adobe.com/cc-shared/assets/sitemap.xml
+https://www.adobe.com/au/cc-shared/assets/sitemap.xml
+https://www.adobe.com/cn/cc-shared/assets/sitemap.xml
```
Comment (3ch023) [inline misc/audit-manifest.txt:3]: where does the list of sitemaps coming from?

**Review by 3ch023 — APPROVED** [review]
approving, although sitemaps list is missing homepage, but we can add it later

File: misc/audit-manifest.txt (hunk 2 above)
Comment (npeltier) [reply misc/audit-manifest.txt:3]: i created it out of the locales we have in our code

Comment (3ch023) [reply misc/audit-manifest.txt:3]: we should probably add a disclamer that these are not all Milo pages, wdyt?

File: misc/audit.mjs (hunk 1 above)
Comment (npeltier) [reply misc/audit.mjs:106]: i think i prefer usage of WCS as blank answer there is also something we should be worried about

File: misc/audit-manifest.txt (hunk 2 above)
Comment (npeltier) [reply misc/audit-manifest.txt:3]: this manifest is here for somebody (me for now) to use. Nothing say this manifest is here to get everything :) (i wait for discussion with business for this)

**Pull request merged**